src/layer/LayerGroup.js

```diff
@@ -77,41 +77,29 @@ export var LayerGroup = Layer.extend({
 	// @method clearLayers(): this
 	// Removes all the layers from the group.
 	clearLayers: function () {
-		for (var i in this._layers) {
-			this.removeLayer(this._layers[i]);
-		}
-		return this;
+		return this.eachLayer(this.removeLayer, this);
 	},
 
 	// @method invoke(methodName: String, …): this
 	// Calls `methodName` on every layer contained in this group, passing any
 	// additional parameters. Has no effect if the layers contained do not
 	// implement `methodName`.
 	invoke: function (methodName) {
-		var args = Array.prototype.slice.call(arguments, 1),
-		    i, layer;
-
-		for (i in this._layers) {
-			layer = this._layers[i];
+		var args = Array.prototype.slice.call(arguments, 1);
 
+		return this.eachLayer(function (layer) {

```

Comment:
I'd keep the simple for in loop in this case because it avoids creating a closure every time (which is bad for performance).

Reply:
@mourner indeed!